code/modules/materials/Mat_Materials.dm

```diff
@@ -489,21 +490,37 @@
 	material_flags = MATERIAL_METAL | MATERIAL_ENERGY
 
 	New()
-		setProperty("density", 65)
+		setProperty("density", 60)
+		setProperty("hard", 20)
 		addTrigger(triggersOnEntered, new /datum/materialProc/soulsteel_entered())
 		return ..()
 
 // Crystals
 /datum/material/crystal
 	material_flags = MATERIAL_CRYSTAL
 	color = "#A3DCFF"
+	New()
```

Comment:
Why is generic crystal doing molitz reaction stuff

Reply:
wuh oh! Unsure how that got in there.